tests/v2/validation/resource/terraform/v1charts/target/main.tf

```diff
@@ -0,0 +1,149 @@
+#Target cluster for v1 charts installation
+terraform {
+  required_version = ">= 0.14"
+  required_providers {
+    rancher2 = {
+      source = "rancher/rancher2"
+    }
+  }
+}
+
+provider "rancher2" {
+  api_url   = var.rancher_api_url
+  token_key = var.rancher_token_key
+}
+
+resource "rancher2_cluster_sync" "v1target-sync" {
+  cluster_id =  var.cluster_id
+}
+
+resource "local_file" "sync_kube_config" {
+    depends_on = [rancher2_cluster_sync.v1target-sync]
```

Comment:
Do you actually need this `depends_on`? Seems like it should wait for the sync by default since you reference it in the `content`